app/src/features/Batches/BatchListItem.tsx

```diff
@@ -62,15 +62,17 @@ const BatchListItem = ({ batch }: BatchListItemType): JSX.Element => {
 
   const getBatchDuration = (batch: Batch) => {
     if (batchEnd) {
-      const start = moment(batch.created_at);
-      const end = moment(batchEnd);
-      const diff = moment.duration(end.diff(start));
-      return `${diff.years() ? `${t("year", { count: diff.years() })} ` : ""}${
-        diff.months() ? `${t("month", { count: diff.months() })} ` : ""
-      }${diff.days() ? `${t("day", { count: diff.days() })} ` : ""}${
-        diff.hours() ? `${t("hour", { count: diff.hours() })} ` : ""
-      }${diff.minutes() ? `${t("minute", { count: diff.minutes() })} ` : ""}${
-        diff.seconds() ? `${t("second", { count: diff.seconds() })} ` : ""
+      const { years, months, days, hours, minutes, seconds } = DateTime.fromISO(
+        batchEnd
+      )
+        .diff(DateTime.fromISO(batch.created_at))
+        .toObject();
```

Comment:
It's not displaying at home. When you call for `diff().toObject()`,  it only gives you the time in miliseconds. You have to specify what data you want to have and you will get an object with all duration units.

```suggestion
      const { years, months, days, hours, minutes, seconds } = DateTime.fromISO(
        batchEnd
      ).diff(DateTime.fromISO(batch.created_at), [
        "years",
        "months",
        "days",
        "hours",
        "minutes",
        "seconds",
      ]);
```